language/stdlib/modules/DesignatedDealer.move

```diff
@@ -41,7 +41,7 @@ module DesignatedDealer {
         let tiers = &mut dealer.tiers;
         let number_of_tiers: u64 = Vector::length(tiers);
         // INVALID_TIER_ADDITION
-        assert(number_of_tiers <= 4, 3);
+        assert(number_of_tiers <= 4, 31);
```

Comment:
We need to be able to update the tiers though - they will probably be different across DDs and currencies

Reply:
Fair, that would be a lot of code duplication. But does `add_tier` need to be public (i.e., can't you initialize the tiers in `publish_designated_dealer_credential`)? That would make it obvious that the tiers are initialized + eliminate some code.

Reply:
yes - you're right. I think the specs are not fully set it stone yet - it may be that they do only want one/two default tiers and then be able to add more. I might leave it for now while I find out more about exactly what the desired usage is.